_includes/manuals/1.10/4.3.5.3_gsstsig.md

```diff
@@ -53,11 +53,11 @@ Note the `\047` is written verbatim, and don't forget the semicolon.  ACLs shoul
 
 #### Proxy configuration
 
-Next, update the proxy configuration file (`/etc/foreman-proxy/settings.d/dns.yml`) with the following settings:
+Next, update the proxy DNS configuration file (`/etc/foreman-proxy/settings.d/dns.yml`) with the following setting:
 
     :dns_provider: nsupdate_gss
+
+And the DNS GSS configuration file (`/etc/foreman-proxy/settings.d/dns_nsupdate_gss.yml`) with:
+
     :dns_tsig_keytab: /etc/foreman-proxy/dns.keytab
     :dns_tsig_principal: foremanproxy/proxy.example.com@EXAMPLE.COM
-    #:dns_key: false
-
-To work correctly, `:dns_key:` must be commented out.
```

Comment:
Ah yes, I think it could still affect it if specified.  Thanks, I've added a new comment.